messages/de-DE.json

```
@@ -138,13 +138,13 @@
         "description": "Wähle das Mitglied, das die Einnahme erhalten hat."
       },
       "recurrenceRule": {
-        "label": "Expense Recurrence",
-        "description": "Select how often the expense should repeat.",
+        "label": "Wiederholung der Ausgabe",
+        "description": "Wähle aus, wie oft die Ausgabe wiederholt werden soll.",
```

Hm, I'm new to spliit, but unless I'm mistaken, this is part of the `"Income"` tree, not of the `"Expense"` tree. So it should probably be "Wiederholung der Einnahme":

```suggestion
        "label": "Wiederholung der Einnahme",
        "description": "Wähle aus, wie oft die Einnahme wiederholt werden soll.",
```

Other than that, everything looks fine :)